src/main/kotlin/com/github/vol0n/utbotcppclion/actions/utils/BuildFuncs.kt

```diff
@@ -0,0 +1,121 @@
+package com.github.vol0n.utbotcppclion.actions
```

Comment:
I suggest to name file `getgRPCMessages` and name functions like `getSettingsContextMessage`